### ISSUE TITLE
[Adaptive Cards> Designer> Select host app]: Screen reader is announcing extra information as 'Current' for all the expanded menu items of select host app combobox.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7918]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25197.1000)
Browser: Edge Dev (107.0.1379.1)
URL: https://adaptivecards.io/designer/
Screen reader: Narrator

**Repro steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through 'Ctrl+ Windows+ Enter key.
3. Navigate to the 'Designer' tab through tab key and activate it.
4. Navigate to the 'Select host app' combobox through tab key and activate it.
5. Navigate to the expanded menu items of 'Select host app' combobox through up/down arrow key.
6. Observe the issue.

**Actual Result:**
On activating the 'Select host app' combo box, screen reader is announcing extra information as 'current' for all the expanded menu items. Screen reader is announcing as '**Bot framework webchat menu item 1 of 10 current'.**

**Expected Result:**
On activating the 'Select host app' combo box, screen reader should not announce any extra information as 'current' with all the expanded menu items. Screen reader should only announce 'selected with the menu items which has already been selected in the combobox.

**Notes:**
Same issue repro with expanded menu items of 'Target Version'.

**User Impact:**
Screen reader user will get impacted as user get irritated while listening the same information with the menu items which create difficulty in tracking the content properly.

MAS Reference link:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104483005/191731132-139edf37-5ad8-495a-bf0d-305004170bd5.mp4

<img width="960" alt="Code Snippet~ Screen reader is announcing current with all the menu items of select host app" src="https://user-images.githubusercontent.com/104483005/191731762-91c8bf0d-f340-4769-8bdb-332239526674.png">

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Closing this issue because it is a duplicate of https://github.com/microsoft/AdaptiveCards/issues/7781.

**Comment by vagpt** [comment]
This issue is working fine on the environment below. hence closing this issue.

Test Environment:
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25330.1000)
Browser Version: 114.0.1807.1
Screen Reader: Narrator
